src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -718,4 +718,24 @@ public boolean hasIndicatedSections(String courseId) throws EntityDoesNotExistEx
         }
         return archivedCourseIds;
     }
+
+    private CourseAttributes validateAndCreateCourseAttributes(String courseId,
+                                                               String courseName, String courseTimeZone)
+            throws InvalidParametersException {
+
+        FieldValidator validator = new FieldValidator();
+        String timeZoneErrorMessage = validator.getInvalidityInfoForCourseTimeZone(courseTimeZone);
+        if (!timeZoneErrorMessage.isEmpty()) {
+            CourseAttributes dummyCourse = CourseAttributes
+                    .builder(courseId, courseName, ZoneId.of("UTC"))
```

Comment:
Can use `Const.DEFAULT_TIMEZONE` to replace `"UTC"`?

Reply:
I think we should remove `Const.DEFAULT_TIMEZONE`. Times are recorded as instants which has no associated timezone, so i don't see how we should have this constant.

Reply:
Anyway it's just a valid dummy timezone so I don't see why we need a constant for it.

Reply:
Ah, my bad. I thought you wanted to use it because you fixed it to `"UTC"` in #8540 that just got merged.